avro-fastserde/src/codegen/java/com/linkedin/avro/fastserde/generated/deserialization/AVRO_1_4/ClasspathTestRecord_SpecificDeserializer_4580844570251215039_2703298965059642600.java

```diff
@@ -1,269 +0,0 @@
-
-package com.linkedin.avro.fastserde.generated.deserialization.AVRO_1_4;
-
-import java.io.IOException;
-import com.linkedin.avro.fastserde.FastDeserializer;
-import org.apache.avro.Schema;
-import org.apache.avro.generic.GenericArray;
-import org.apache.avro.io.Decoder;
-import org.apache.avro.util.Utf8;
-
-public class ClasspathTestRecord_SpecificDeserializer_4580844570251215039_2703298965059642600
```

Comment:
It seems like we're replacing this code-generated class with another one by altering the test. Can we check-in the replacement for this class as part of this first commit? The goal is to be able to easily visualize the diff to the codegen output in the third commit.

Reply:
Yes, I decided to change the test name to make it more appropriate for the actual case. Please, check the latest push. The first commit contains the refactored test with codegen classes, which don't compile due to the semicolon issue. The second commit is the simple fix, and the last one are the fixed codegen classes.